constants/objectives/condition_bits.py

```diff
@@ -29,9 +29,14 @@
     NameBit("Floating Cont. Escape", event_bit.FINISHED_FLOATING_CONTINENT),
     NameBit("Gau's Father's House", event_bit.RECRUITED_SHADOW_GAU_FATHER_HOUSE),
     NameBit("Imperial Camp", event_bit.FINISHED_IMPERIAL_CAMP),
+    # NameBit("Kefka's Tower Ambush", event_bit.DEFEATED_INFERNO),
```

Comment:
i believe the id of the event is based on the index of this array, probably will need to throw these at the bottom